manifests/deployment.yml

```diff
@@ -104,11 +104,11 @@ spec:
                 secretKeyRef:
                   name: semesterly-secrets
                   key: SOCIAL_AUTH_AZURE_TENANT_SECRET
-            - name: JWT_AUTH_SECRET
+            - name: STUDENT_SIS_AUTH_SECRET
               valueFrom:
                 secretKeyRef:
                   name: semesterly-secrets
-                  key: JHU_JWT_AUTH_SECRET
+                  key: STUDENT_SIS_AUTH_SECRET    

```

Comment:
is this going to break anything??

Reply:
I have to add in changes from cs310 branch and this will be reversed.